feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailScreen.kt

```diff
@@ -293,11 +293,7 @@ fun MovieDetailContent(
     }
     Text(
       modifier = Modifier.padding(horizontal = 16.dp),
-      style = TextStyle(
-        fontWeight = FontWeight.Bold,
-        fontSize = 16.sp,
-        color = Color.Black,
-      ),
+      style = MaterialTheme.typography.bodyLarge.copy(fontWeight = FontWeight.Bold),
```

Comment:
**Pro Tip:**  💅💅💅 Please do not change something that is not related to this Issue/PR. Maybe someone else is working on this screen at the same time and it might led to conflict.